### ISSUE TITLE
Simplify getting data for `datashader_dashboard`

### ISSUE THREAD

**Issue by maximlt** [pyviz-topics/examples#302]
Getting data for `datashader_dashboard` is convoluted, it required `anaconda-project` as a dependency to run `prepare` on other projects and uses intake to wire everything. I don't think this dependence on other projects is very useful in the end, we should simplify it.

**Comment by jbednar** [comment]
Agreed. Too messy; even I forget how it works, and I have some vague idea that I might have written it.
